app/app_test.go

```diff
@@ -398,6 +400,46 @@ func TestInfoSync(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestRelayConnections(t *testing.T) {
```

Comment:
Could be good to add a negative case to show it failed before this

Reply:
ok, added 1025th connection which should error